backend/accounts/middleware.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, get_response):
 
     def __call__(self, request):
 
-        if request.path in self.EXCLUDED_PATHS or request.path.startswith('/admin'):
+        if request.path in self.EXCLUDED_PATHS or request.path.startswith('/admin') or request.path.startswith('/api/yugioh'):
```

Comment:
It''s not a problem, but try not to have changes that are not related to this PR, especially at work.